### ISSUE TITLE
Tiller install error: configmaps is forbidden

### ISSUE THREAD

**Issue by snpdev** [microsoft/RockPaperScissorsLizardSpock#2]
On running  .\Add-Tiller.ps1 from my local Powershell, I have the following error:
```
------------------------------------------------------------
Installing Helm
------------------------------------------------------------
Error: configmaps is forbidden: User "system:serviceaccount:kube-system:default" cannot list resource "configmaps" in API group "" in the namespace "kube-system"
```
Error occurs on the command `helm list -q`

I'm investigating this now, but if anyone wants to chime in with a solution, please do.

**Comment by snpdev** [comment]
I found the solution here https://github.com/helm/helm/issues/5100#issuecomment-533787541.

**Comment by cdemiguel** [comment]
Hello, 
i am glad to help you,
In a new installation this error should not occur, it seems that you do not have permissions to deploy tiller with your serviceaccount,
regards,

**Comment by OriolBonjoch** [comment]
Hi @snpdev 

If that error happens on the `helm ls` line, is a validation before installing tiller. It allows to know if tiller has to be installed as client-only or not.
Besides the stderror message there was a problem with the execution or it worked fine?
If not looks like we might to move the service account to be `tiller` as your solution suggested.

Thanks for the report and the solution!

Regards,

**Comment by snpdev** [comment]
@OriolBonjoch, .\Add-Tiller.ps1 execution failed at the error noted in https://github.com/microsoft/RockPaperScissorsLizardSpock/issues/2#issue-519406885.

Hat tip to [m-abramovich](https://github.com/m-abramovich) for the solution.

**Comment by olgamarti** [comment]
The issue is fixed in PR #11 .
